src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -367,6 +440,47 @@ export class RuleEditor extends LinkedComponent {
                 </Section.Container>
               ))
             }
+            <Section.Container collapsable={false}>
+              <Section.Header>{t('rules.flyouts.ruleEditor.actions.actions')}</Section.Header>
+              <ToggleBtn
+                value={formData.actionEnabled}
+                onChange={this.onActionToggle} >
+                {formData.actionEnabled ? t('rules.flyouts.ruleEditor.actions.on') : t('rules.flyouts.ruleEditor.actions.off')}
+              </ToggleBtn>
+              {
+                formData.actionEnabled && <p>{t('rules.flyouts.ruleEditor.actions.sendEmailOnTrigger')}</p>
+              }
+              {
+                actionLinks.map((action, idx) => (
+                  <div key={formData.actions[idx].key}>
+                    <Section.Content>
+                      {
+                        formData.actionEnabled &&
+                        <div>
+                          <FormGroup>
+                            <FormControl
+                              type="text"
+                              className="long"
+                              onKeyPress={this.onAddEmail(action.emailLink)}
+                              link={this.newEmailLink}
+                              placeholder={t('rules.flyouts.ruleEditor.actions.enterEmail')} />
+                          </FormGroup>
+                          <PillContainer
```

Comment:
I would name `PillContainer` something else (maybe `PillFormControl`). The reason is that `container` has a particular meaning for a react component name - specifically that it is a wrapper connecting a presentational component with the redux store or some other external data sources.

Reply:
Renamed.